unyt/unit_systems.py

```diff
@@ -178,13 +199,16 @@ def __repr__(self):
 planck_unit_system["energy"] = "E_pl"
 planck_unit_system["charge_cgs"] = "q_pl"
 
-#: A CGS unit system with Ampere (SI-like E&M units)
-cgs_ampere_unit_system = UnitSystem('cgs-ampere', 'cm', 'g', 's',
-                                    current_mks_unit='A')
-cgs_ampere_unit_system["energy"] = "erg"
-cgs_ampere_unit_system["specific_energy"] = "erg/g"
-cgs_ampere_unit_system["pressure"] = "dyne/cm**2"
-cgs_ampere_unit_system["force"] = "dyne"
-cgs_ampere_unit_system["magnetic_field_cgs"] = "gauss"
-cgs_ampere_unit_system["charge_cgs"] = "esu"
-cgs_ampere_unit_system["current_cgs"] = "statA"
+#: The internal base unit system: CGS with extra SI units
```

Comment:
Possibly crazy idea: why don't we just use SI?

That'll make integrating with unyt back inside yt slightly harder but that's probably worth the effort to make `unyt` easier to use for a random non-astronomer user. You'd be able to work with CGS units pretty easily anyway.

Not something that needs to happen in this PR of course.

Reply:
To be honest, I think that this is what we should do. I suspect that integrating back into yt will actually not be that hard--it may actually be all internal to unyt. We'd have to change the default base units and the `unit_lookup_table`, and maybe one or two other things.